src/Tribe/Main.php

```diff
@@ -944,7 +944,7 @@ public function init() {
 			$this->singular_event_label_lowercase             = tribe_get_event_label_singular_lowercase();
 			$this->plural_event_label_lowercase               = tribe_get_event_label_plural_lowercase();
 
-			$this->post_type_args['rewrite']['slug']            = $rewrite->prepare_slug( $this->rewriteSlugSingular, self::POSTTYPE, false );
+			$this->post_type_args['rewrite']['slug']          = $rewrite->prepare_slug( $this->rewriteSlugSingular, self::POSTTYPE, false );
```

Comment:
* :warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 10 spaces
* :no_entry_sign: Object property "$rewriteSlugSingular" is not in valid snake_case format, try "$rewrite_slug_singular"

<sub>hash: 19e2434a0f39188f40b07e8e08191b42</sub>